packages/combo-box/test/scrolling.test.js

```diff
@@ -86,26 +86,13 @@ describe('scrolling', () => {
       expect(selectedItemRect.bottom).to.be.at.most(overlayRect.bottom + 1);
     }
 
-    it('should make selected item visible after open', async () => {
+    it.skip('should make selected item visible after open', async () => {
```

Comment:
Why do we skip this test rather than completely remove it?

Reply:
Removed. It was an oversight on my part from the first edition. Thanks for pointing it out.